### DISCUSSION TITLE
Use original Microsoft icons for weather

### DISCUSSION THREAD

**Discussion by novatiberium** [valinet/ExplorerPatcher#939] [Ideas]
In EP's weather implementation, it uses icons from Google. If possible, would it be possible to go into the bitmaps of the Windows 10 implementation and take the icons from there?

**Comment by valinet** [comment]
Do you know where those are? We’d also need a complete mapping from Google icon to Microsoft icon. And it would kind of break the consistency.

Btw, why does it have to be Microsoft? Google’s icons are quite good as well. Why brand it Microsoft when Google actually provides it? Is it because Google’s set uses day icons for the night time as well? That can be fixed separately, attempting to manually clear out some pixels from the sun to turn it into a moon for the 3 or so icons that feature a sun. At some point, I plan to do that as well.

**Reply by valinet** [reply]
Btw, it would be nice if people would also pick tasks to work on regarding this from here: https://github.com/valinet/ExplorerPatcher/discussions/755.

Some plans are already laid out, but unfortunately I can do so much in 24 hours. The tasks there are not that hard, and they’re not C, they just take some effort and time to polish and get right.
